src/components/Proposal/Proposal.jsx

```diff
@@ -547,6 +548,15 @@ const Proposal = React.memo(function Proposal({
                   {(isVoteActive || isVotingFinished) && !legacytoken && (
                     <ChartsLink token={fullToken} />
                   )}
+                  {isVoteActive && getVotesReceived(voteSummary) > 0 && (
```

Comment:
```suggestion
                  {getVotesReceived(voteSummary) > 0 && (
```
Here, IMO, we should display it for every proposal that received votes.